engine/access/state_stream/api_test.go

```diff
@@ -0,0 +1,121 @@
+package state_stream
+
+import (
+	"bytes"
+	"context"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/ipfs/go-datastore"
+	dssync "github.com/ipfs/go-datastore/sync"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/onflow/flow-go/engine/common/rpc/convert"
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/common/testutils"
+	"github.com/onflow/flow-go/module/blobs"
+	"github.com/onflow/flow-go/module/executiondatasync/execution_data"
+	storagemock "github.com/onflow/flow-go/storage/mock"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+type Suite struct {
+	suite.Suite
+
+	headers *storagemock.Headers
+	seals   *storagemock.Seals
+	results *storagemock.ExecutionResults
+}
+
+func TestHandler(t *testing.T) {
+	suite.Run(t, new(Suite))
+}
+
+func (suite *Suite) SetupTest() {
+	rand.Seed(time.Now().UnixNano())
+	suite.headers = new(storagemock.Headers)
```

Comment:
if you create these with the `storagemock.NewHeaders` method, it will `AssertExpectations` automatically when the test ends